Valet/VALValet.m

```diff
@@ -167,7 +167,12 @@ + (VALValet *)_sharedValetForValet:(VALValet *)valet;
 
 #pragma mark - Initialization
 
-- (instancetype)initWithIdentifier:(NSString *)identifier accessibility:(VALAccessibility)accessibility;
+- (nullable instancetype)init NS_UNAVAILABLE;
+{
+    VALCheckCondition(NO, nil, @"Use a designated initializer");
+}
+
+- (nullable instancetype)initWithIdentifier:(NSString *)identifier accessibility:(VALAccessibility)accessibility;
```

Comment:
good catch! annoying that the compiler doesn't consider this an error. I guess if the .h declares it the .m doesn't need it... but that sucks.